newscoop/install/Resources/sql/campsite_core.sql

```diff
@@ -294,8 +294,6 @@ CREATE TABLE `Articles` (
   `section_id` INT DEFAULT NULL,
   PRIMARY KEY (`IdPublication`,`NrIssue`,`NrSection`,`Number`,`IdLanguage`),
   UNIQUE KEY `IdPublication` (`IdPublication`,`NrIssue`,`NrSection`,`IdLanguage`,`Name`),
-  UNIQUE KEY `Number` (`Number`,`IdLanguage`),
-  UNIQUE KEY `other_key` (`IdPublication`,`NrIssue`,`NrSection`,`IdLanguage`,`Number`),
```

Comment:
This should stay, as without it there is no unique key for Number and IdLanguage.